services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusyhteystietoEntity.kt

```diff
@@ -57,7 +57,28 @@ class HakemusyhteystietoEntity(
             null
         )
 
-    fun toDomain() = Hakemusyhteystieto(id, tyyppi, rooli, nimi, sahkoposti, puhelinnumero, ytunnus)
+    fun toDomain() =
+        Hakemusyhteystieto(
+            id = id,
+            tyyppi = tyyppi,
+            rooli = rooli,
+            nimi = nimi,
+            sahkoposti = sahkoposti,
+            puhelinnumero = puhelinnumero,
+            ytunnus = ytunnus,
+            yhteyshenkilot =
+                yhteyshenkilot.map { yhteyshenkilo ->
+                    Hakemusyhteyshenkilo(
```

Comment:
Here is the same as with ApplicationEntity.yhteystiedot above - a session/transasction is needed. Maybe we could add some comments on these `toDomain()` functions about this?

Reply:
I think one should assume that any method in an entity can cause `LazyInitializationException`s, with the exception of simple values.